scripts/dts/edtlib.py

```diff
@@ -1255,6 +1261,10 @@ def _check_binding(binding, binding_path):
             _err("enum in {} for property '{}' is not a list"
                  .format(binding_path, prop_name))
 
+        if "const" in options and isinstance(options["const"], list):
```

Comment:
Can get dictionaries from YAML as well. Listing the allowed stuff instead of the forbidden stuff might be better, like this:

```python
if "const" in options and not isinstance(options["const"], (int, str)):
    _err(...)
```

(`isinstance` can take a tuple of types.)

Reply:
Fixed.